src/Raven.Server/Documents/Sharding/Handlers/Processors/Subscriptions/ShardedSubscriptionsHandlerProcessorForTrySubscription.cs

```diff
@@ -50,7 +50,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandl
                 writer.WriteEndArray();
                 writer.WriteComma();
                 writer.WritePropertyName("Includes");
-                DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Egor, DevelopmentHelper.Severity.Major, "https://issues.hibernatingrhinos.com/issue/RavenDB-16279");
+                DevelopmentHelper.ShardingToDo(DevelopmentHelper.TeamMember.Egor, DevelopmentHelper.Severity.Minor, "need to either add missing incldues, or just return nulls for them here. https://issues.hibernatingrhinos.com/issue/RavenDB-16279");
```

Comment:
I would say we need to add missing includes here

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-20024/Add-missing-includes-for-ShardedSubscriptionsHandlerProcessorForTrySubscription